<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QWidget" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="choix1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Choix 1 :</string>
   </property>
   <widget class="QGroupBox" name="couleur1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Couleur</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="jaune1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Jaune</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rouge1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rouge</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bleu1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bleu</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="forme1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Forme</string>
    </property>
    <widget class="QRadioButton" name="carre1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Carré</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="triangle1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Triangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rond1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rond</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="choix2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Choix 2 :</string>
   </property>
   <widget class="QGroupBox" name="couleur2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Couleur</string>
    </property>
    <widget class="QRadioButton" name="jaune2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Jaune</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rouge2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rouge</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bleu2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bleu</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="forme2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Forme</string>
    </property>
    <widget class="QRadioButton" name="carre2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Carré</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="triangle2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Triangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rond2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rond</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="choix3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Choix 3 :</string>
   </property>
   <widget class="QGroupBox" name="couleur3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Couleur</string>
    </property>
    <widget class="QRadioButton" name="jaune3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Jaune</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rouge3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rouge</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bleu3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bleu</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="forme3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Forme</string>
    </property>
    <widget class="QRadioButton" name="carre3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Carré</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="triangle3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Triangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rond3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rond</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="marche">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Marche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arret">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrêt</string>
   </property>
  </widget>
  <widget class="QLabel" name="optiondetri">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Options de tri :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbpiece1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nomcompteur1">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de pièces triées :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbpiece2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>280</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nomcompteur2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de pièces triées :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbpiece3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>460</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nomcompteurx3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>430</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de pièces triées :</string>
   </property>
  </widget>
  <widget class="QWidget" name="video" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>411</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
